packages/react-tabster/src/hooks/useNavigationAttribute.ts

```diff
@@ -0,0 +1,19 @@
+import { RefObject, useEffect, useRef } from 'react';
+import { KEYBOARD_NAV_ATTRIBUTE } from '../symbols';
+import { useTabster } from './useTabster';
+
+export const useNavigationAttribute = (): RefObject<HTMLElement> => {
+  const tabster = useTabster();
+  const ref = useRef<HTMLElement>(null);
+  useEffect(() => {
+    if (tabster) {
+      const setKeyboardNavAttribute = (nextIsKeyboard: boolean) => {
+        ref.current?.setAttribute(KEYBOARD_NAV_ATTRIBUTE, nextIsKeyboard.toString());
+      };
+      setKeyboardNavAttribute(tabster.keyboardNavigation.isNavigatingWithKeyboard());
+      tabster.keyboardNavigation.subscribe(setKeyboardNavAttribute);
+      return () => tabster.keyboardNavigation.unsubscribe(setKeyboardNavAttribute);
+    }
```

Comment:
We had a discussion about this part and we don't have a conclusion there yet.

From my POV, `tabster` is not the best solution in this case currently as it's not modularized well 😥 I measured this locally, import of this hook costs currently 52KB of minified code:

![bundle size report for useNavigationAttribute, useNavigationAttribute minified is 52KB](https://user-images.githubusercontent.com/14183168/117438298-aca9ce80-af31-11eb-88eb-ce82adc02802.png)